docs(v2): emphasize subset of markdown supported languages (#4725)
facebook/docusaurus: website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

};
```

> By default, Docusaurus comes with this subset of [commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js). Note that some popular languages like Java, C#, or PHP are not included in this list.

Reviewer: ```suggestion
By default, Docusaurus comes with a subset of [commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js). 

:::caution

Some popular languages like Java, C#, or PHP are not enabled by default.

:::
```
